app/models/optional_field.rb

```diff
@@ -0,0 +1,13 @@
+class OptionalField < ApplicationRecord
+  belongs_to :user
+
+  validates :field, presence: true, length: { maximum: 30 }
```

Comment:
this could be `label`, or `name`. So that the pair is `label -> value`, or `name -> value`

Reply:
Done